### PR TITLE
CCSO-10 Added onboarding diagram DRAFT ONLY

### PR DESCRIPTION
**Pull request by jbeltranatl**
5 step process diagram added

### CODE REVIEW

File: doc/gov/chapters/chapter02.md [hunk 1]
```diff
@@ -34,13 +34,8 @@ As an open-source community organization, CNTT has developed the following proce
...
   
-  - **Overall, the process is as follows**
-  - 1. Request Membership
-  - 2. Review General Information packet
-  - 3. Agree to Terms
-  - 4. Join the Community
-  - 5. Begin Contributing
-  
+  - **Diagram**
+<p align="middle"><img src="../figures/onboarding_proc.png" alt="onboarding_proc" title="Onboarding Process" width="60%"/></p>
```
Comment (CsatariGergely) [inline doc/gov/chapters/chapter02.md]: The figure is not part of the pr. Please add it.

File: doc/gov/chapters/chapter02.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [<< Back](../)
 
 # 2. Onboarding
-<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_sdc.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
 * [2.1 Introduction](#2.1)
-* [2.2 Membership](#2.2)
+* [2.2 Request Membership](#2.2)
```
Comment (CsatariGergely) [inline doc/gov/chapters/chapter02.md]: Is it possible to request a membership to a task force?

File: doc/gov/chapters/chapter02.md (hunk 1 above)
Comment (jbeltranatl) [reply doc/gov/chapters/chapter02.md]: Gergely, yea. Well, it's on a separate one. See #656

File: doc/gov/chapters/chapter02.md (hunk 2 above)
Comment (jbeltranatl) [reply doc/gov/chapters/chapter02.md]: Gergely, technically yes. "task force" by definition really just means the org was assembled to make a special thing/'task' happen, and it typically has a short-term connotation. But it's still an organization. But I take your point as we need some introductory language that says anyone/org is free to join because it is an open source community...etc...so folks don't get confused.

File: doc/gov/chapters/chapter02.md [hunk 3]
```diff
@@ -34,13 +34,8 @@ As an open-source community organization, CNTT has developed the following proce
   
   - **What happens once I get onboarded?** Once you have reviewed the CNTT onboarding information and have agreed to the perquisite terms and agreements associated with participating and contributing to the Task Force, and you have completed onboarding and now it will be your opportunity to join desired work streams. For an initial period after onboarding, you will be assigned a volunteer mentor from the Task Force to help you get up to speed and contributing quickly.
```
Comment (pgoyal01) [inline doc/gov/chapters/chapter02.md]: Should the description of what happens once "I have onboarded" include the onboarding steps or simply state, "Once you have completed the onboarding process, you will be able to join work streams. If you want to retain current wording, you still need to change (after all the "and"s) the phrase, "... and now it will be your opportunity to join desired work streams" to, say, "...  it will be your opportunity to join desired work streams."

Typo, "perquisite" ---> "prerequisite"

Comment (jbeltranatl) [reply doc/gov/chapters/chapter02.md]: Pankaj. This is the getting started. It is not the onboarding packet itself. But point taken. I would like to get all draft versions of doc in first before we start dickering over fine points.

File: doc/gov/chapters/chapter02.md (hunk 1 above)
Comment (CsatariGergely) [reply doc/gov/chapters/chapter02.md]: > 
> 
> Gergely, yea. Well, it's on a separate one. See #656

Can you merge this two pr-s?

Comment (rabi-abdel) [reply doc/gov/chapters/chapter02.md]: @CsatariGergely @jbeltranatl I have moved the diagram into this PR. will resolve now.

**Comment by rabi-abdel** [comment]
@jbeltranatl @CsatariGergely I have moved the diagram from the previous Pull Request into here as per @CsatariGergely request.
